recipes/plasmapy/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "plasmapy" %}
+{% set version = "0.1.dev968" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "f127782615810e6f45f39f63a51b7eda76a9fe0286b4aef54f265cc744e946a6" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+  skip: True  # [py2k]
```

Comment:
Could you put the python version requirement into the deps and make this a noarch package? I'm assuming that this doesn't compile anything.

Reply:
Not currently, no, so I've changed that. Thanks.